cancer_ai/validator/competition_manager.py

```diff
@@ -140,12 +141,16 @@ async def get_miner_model(self, chain_miner_model: ChainMinerModel):
 
     async def sync_chain_miners_test(self):
         """Get registered mineres from testnet subnet 163"""
-
         hotkeys_with_models = {
-            "5Fo2fenxPY1D7hgTHc88g1zrX2ZX17g8DvE5KnazueYefjN5": ModelInfo(
-                hf_repo_id="safescanai/test_dataset",
-                hf_model_filename="model_dynamic.onnx",
-                hf_repo_type="dataset",
+            "hf_OgEeYLdTgrRIlWIdmbcPQZWTtdKfSwwddsavDfO": ModelInfo(
```

Comment:
add comments which is working and which aren't

Reply:
done